tests/__init__.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import annotations
+
+import multiprocessing
+
+try:
+    multiprocessing.set_start_method("spawn")
+except RuntimeError:
```

Comment:
I'm still not quite sure about this.  There are ongoing direct discussions.  I think this is just making this code a no-op.